src/java.desktop/windows/native/libawt/windows/awt_List.cpp

```diff
@@ -171,16 +171,6 @@ void AwtList::Reshape(int x, int y, int w, int h)
 {
     AwtComponent::Reshape(x, y, w, h);
 
```

Comment:
I would also remove the blank line in the end of the function, it's uncommon to end a function with a blank line.